lib/test/Value_Type_Test.thy

```diff
@@ -5,11 +5,13 @@
  *)
 
 theory Value_Type_Test
-imports Lib.Value_Type
+  imports
+    Lib.Value_Type
+    "Word_Lib.WordSetup"
 begin
 
 (*
-   Define a type synonym from a term that evaluates to a numeral.
+   Define a type synonym from a term of type nat or int that evaluates to a numeral.
```

Comment:
same comment as the comment about negative values

Reply:
Happy to add that